public/ng-views/maps.html

```diff
@@ -19,23 +22,23 @@
         <small>{{ user.name }}</small>
       </i>
     </span>
-  </span>
+  </span> -->
 </div>
 
 
-<div class="topnav right">
+<!-- <div class="topnav right">
   <div class="fa-stack" ng-click="saveCombo=true">
     <i class="fa fa-circle fa-stack-2x"></i>
     <i class="fa fa-arrow-left fa-stack-1x fa-inverse"></i>
-  </div>
-  <div class="fa-stack fa-lg" ng-click="dotSelect = !dotSelect; updateTargetFeature(); overview=false; tabnav=null">
+  </div> -->
+  <!-- <div class="fa-stack fa-lg" ng-click="dotSelect = !dotSelect; updateTargetFeature(); overview=false; tabnav=null">
     <i class="fa fa-circle fa-stack-2x"></i>
     <i class="fa fa-bus fa-stack-1x fa-inverse"></i>
-  </div>
+  </div> -->
 
```

Comment:
I hard coded these select dots but the fa-stack component with the dots may not be the best move for the environment you are dealing with - it doesn't handle overflows and alignment because it is a font not a div, so in such a tight space: 
![image](https://cloud.githubusercontent.com/assets/6053396/8777459/1b65acb0-2ec6-11e5-9ab6-bf9aaf6a5e50.png)